### PR TITLE
[NFC] Update codeowners

### PR DESCRIPTION
**Pull request by romanovvlad**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -95,18 +76,18 @@ xpti/ @tovinkere @andykaylor
...
+llvm/lib/Support/PropertySetIO.cpp @mlychkov @AlexeySachkov
+llvm/unittests/Support/PropertySetIOTest.cpp @mlychkov @AlexeySachkov
+llvm/lib/Support/Base64.cpp @mlychkov @AlexeySachkov
 
 # Clang offload tools
-clang/tools/clang-offload-bundler/ @kbobrovs @sndmitriev
-clang/tools/clang-offload-wrapper/ @sndmitriev @kbobrovs
+clang/tools/clang-offload-bundler/ @mlychkov @sndmitriev
+clang/tools/clang-offload-wrapper/ @sndmitriev @mlychkov
 clang/tools/clang-offload-deps/ @sndmitriev
```
Comment (bader) [inline .github/CODEOWNERS:91]:
```suggestion
clang/tools/clang-offload-deps/ @sndmitriev @mlychkov 
```
@mlychkov?

**Review by bader — DISMISSED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (Fznamznon) [reply .github/CODEOWNERS:91]: I heard Mikhail is on vacation.
Tagging @AlexeySachkov .

Comment (AlexeySachkov) [reply .github/CODEOWNERS:91]: I'm not sure that either of us has enough expertise in clang offload tools to replace Konst by oneself, but probably if we both of us were added, then it should be fine.

Comment (romanovvlad) [reply .github/CODEOWNERS]: Added Mikhail.

Comment (bader) [reply .github/CODEOWNERS]: I think Alexey suggests adding @AlexeySachkov as well.

Comment (romanovvlad) [reply .github/CODEOWNERS]: Added Alexey as well.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -95,19 +76,19 @@ xpti/ @tovinkere @andykaylor
...
+llvm/lib/Support/PropertySetIO.cpp @mlychkov @AlexeySachkov
+llvm/unittests/Support/PropertySetIOTest.cpp @mlychkov @AlexeySachkov
+llvm/lib/Support/Base64.cpp @mlychkov @AlexeySachkov
 
 # Clang offload tools
-clang/tools/clang-offload-bundler/ @kbobrovs @sndmitriev
-clang/tools/clang-offload-wrapper/ @sndmitriev @kbobrovs
-clang/tools/clang-offload-deps/ @sndmitriev
+clang/tools/clang-offload-bundler/ @mlychkov @sndmitriev
+clang/tools/clang-offload-wrapper/ @mlychkov @sndmitriev
```
Comment (bader) [inline .github/CODEOWNERS:90]: @AlexeySachkov, do you want to be added for bundler and wrapper tools as well?
```suggestion
clang/tools/clang-offload-bundler/ @mlychkov @sndmitriev @AlexeySachkov
clang/tools/clang-offload-wrapper/ @mlychkov @sndmitriev @AlexeySachkov
```

Comment (AlexeySachkov) [reply .github/CODEOWNERS:90]: Yes, I think I can be added as a third owner to all `clang-offload-` tools we have

**Review by bader — APPROVED** [review]

**Review by AlexeySachkov — APPROVED** [review]
Lines where I was added as a code owner look good to me

**Review by olegmaslovatintel — APPROVED** [review]
@mlychkov is not available atm

**Review by sndmitriev — APPROVED** [review]

**Pull request merged**